pkg/oc/cli/cmd/startbuild.go

```diff
@@ -462,6 +454,29 @@ func (o *StartBuildOptions) Run() error {
 	return nil
 }
 
+func (o *StartBuildOptions) streamBuildLogs(build *buildapi.Build) error {
+	opts := buildapi.BuildLogOptions{
+		Follow: true,
+		NoWait: false,
+	}
+	var err error
+	for i := 0; i < streamLogAttempts; i++ {
+		rd, logErr := o.BuildLogClient.Logs(build.Name, opts).Stream()
+		if logErr != nil {
+			err = ocerrors.NewError("error getting logs").WithCause(logErr)
```

Comment:
unclear why, but the `ocerrors.IsTimeoutErr` check always returned false, so I removed it.